### DISCUSSION TITLE
missing thermostat local termperature in Homeasssitant

### DISCUSSION THREAD

**Discussion by marrog86** [Koenkk/zigbee2mqtt#13410] [Devices]
Hello, 

i am using different termostats:
- Moes [BRT-100-TRV]
- Liedl [368308_2010]

in Homeasssitant the auto discovery only recognizes the local_temperature_callibration, but the local_temperature sensor entity which would be more interesting for homeassistant is not showing up. 

This is for both devices the same. What can i do? I want to use these sensors for showing up when it becomes hot outside and windows should be closed. 

Accessing attributes in Homeasssitant scripts and yaml works, but is more complicated since there is no entity whicht could be selected directly 

Thanks
Martin

**Comment by timhere** [comment]
I'm also experiencing the same issue, I can see the `local_temperature` property is exposed by Zigbee2MQTT, and I can see `local_temperature` as a state attribute, but for some reason this particular property does not seem to be mapped to an entity in HA.

It appears we might be able to create a new sensor using the [Template](https://www.home-assistant.io/integrations/template) integration, but this feels a little messy when the data is right there!

**Comment by Quintenvw** [comment]
I just got a couple Aqara w600's and I had the same issue.
Apparently this is because `local_temperature` it considered an attribute of the climate entity instead of a device sensor.
You can use a template to turn the attribute into a sensor that you can use in Home Assistant like this:
```yaml
{{ state_attr('climate.bathroom_thermostat_radiator_valve', 'current_temperature') }}
```
